README.md

```diff
@@ -14,10 +14,63 @@ For installation instructions, please see our [install manual][install]
 
 ## Set up your Dev environment
 
-1. Make sure you have Postgresql installed
-1. Clone this repo
-1. Run `bin/setup`
-1. Start the app with `foreman start`
+1. Make sure you have PostgreSQL and Redis installed
+2. Clone this repo
+3. Run `bin/setup`
+4. Start the app with `foreman start`
+
+## Set up your Dev environment using Docker
+
+You can also set up your local development environment without having to install
+any outside dependencies. For this, use the Docker Compose configuration that's
+available in this repository. This configuration automatically mounts your
+local code inside a container, so that you can make changes and commit your
+code as if you were running a local development server without Docker.
+
+First, install Docker for your workstation: https://www.docker.com/community-edition
+
+Then, use `docker-compose` commands to set up a local environment using Docker
+and run the development server:
+
+```
+$ docker-compose run web rails db:create
```

Comment:
Would it be possible to just run `docker-compose run web rails bin/setup`? That way you automaticly load seeds etc. I then don't have to keep in my mind that if I change something in my bin/setup I also need to change this readme.

Reply:
Yeah you can run bin/setup. Check out the new README for new instructions. We can put `bin/setup` in there. However, you don't want to run `bin/setup` if you want to test the onboarding.

Reply:
@michiels I think it would make sense to in that case create two users in the seeds. `user@example.com` and `onboarding@example.com`. That way you can always run bin/setup without having to think about it.

Reply:
I think the onboarding in Intercity doesn't fire when `User.count >= 1` so that wouldn't really solve the problem. But it's okay for having people figure this out for themselves. Regular Rails devs will understand that they can `bin/setup` or just run the tasks manually.

Reply:
@michiels Ok 👍 But do you then agree with me that it's best to keep the most sane default in there (being `bin/setup`, because I can then assume that a beginner wants to run the project they run that and everything works ™️ )? We can add an extra header in there on how to test the onboarding for example.

Reply:
Yep having `bin/setup` as starting point is fine for me.